src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/SRCDemoWidgetViewImpl.java

```diff
@@ -55,10 +55,10 @@ public void setSynAlertWidget(IsWidget w) {
 	@Override
 	public void setDemoVisible(boolean visible) {
```

Comment:
Is this still in here?!  You could delete SRCDemoWidget (and view and view impl) and remove the reference from the ComingSoon page in this PR if you'd rather